src/library/slices/GoogleMap/GoogleMap.storydata.ts

```diff
@@ -50,3 +59,16 @@ export const GoogleMapWithUpdatedLinkFormat = {
   ...Common,
   link_url: 'https://maps.app.goo.gl/fwHnJrT7RfAQQ3FN7',
 };
+
+export const OneNetworkMapLink = {
+  title: OneNetworkCommon.title,
+  link_title: OneNetworkCommon.link_title,
+  link_url: OneNetworkCommon.link_url,
+  iframe_html: OneNetworkCommon.iframe_html,
+  allowCookies: true,
+};
+
+export const OneNetworkMapIncorrectOrgDetailsLink = { ...OneNetworkMapLink };
```

Comment:
You can spread another object within the same object, rather than overwriting individual object keys. 

Also, this no longer shows the Invalid map alert message as the component no longer checks for the organisation id. I have amended the URL in the below suggestion to `apis-gb`, rather than `api-gb` so it fails. You could even make it more explicit by changing it to something like `incorrect-url.one.network` 

```suggestion
export const OneNetworkMapIncorrectOrgDetailsLink = {
  ...OneNetworkMapLink,
  ...{
    title: 'One Network Map (Incorrect Organisation IDs)',
    iframe_html:
      '<iframe src="https://apis-gb.one.network/embedded/?options=%7B%22organisationID%22%3A2248" title="Causeway one.network map" style="width: 800px; height: 559px; border: none;"></iframe>',
  },
};
```